packages/react-components/lib/simple-info.tsx

```diff
@@ -89,14 +89,12 @@ export const SimpleInfo = (props: SimpleInfoProps): JSX.Element => {
     value,
     className,
     disabled,
-    wrap,
   }: SimpleInfoData<DataValueTypePrimitive>) => (
     <>
       <Typography className={classes.displayName} variant="body1">
         {name}
       </Typography>
       <Typography
-        noWrap={!wrap}
         variant="body1"
```

Comment:
Just checking, isn't this affecting the task-summary-details?

Reply:
Restored it to the original form.